client/client-multi/client-cpp/Modules/FindBotan.cmake

```diff
@@ -13,122 +13,60 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 # -*- mode: cmake; -*-
 # - Try to find libbotan include dirs and libraries
 # Usage of this module as follows:
 # This file defines:
 # * BOTAN_FOUND if protoc was found
 # * BOTAN_LIBRARY The lib to link to (currently only a static unix lib, not
-# portable) 
+# portable)
 # * BOTAN_INCLUDE The include directories for libbotan.
 
-message(STATUS "FindBotan check")
-IF (NOT WIN32)
-  include(FindPkgConfig)
-  if ( PKG_CONFIG_FOUND )
-
-     pkg_check_modules (PC_BOTAN botan-1.11)
+message(status "FindBotan check")
 
-     set(BOTAN_DEFINITIONS ${PC_BOTAN_CFLAGS_OTHER})
+if(NOT WIN32)
+  include(FindPkgConfig)
+  if(PKG_CONFIG_FOUND)
+     pkg_check_modules(BOTAN botan-1.11)
   endif(PKG_CONFIG_FOUND)
-endif (NOT WIN32)
-
-#
-# set defaults
-SET(_botan_HOME "/usr/local")
-SET(_botan_INCLUDE_SEARCH_DIRS
-  ${CMAKE_INCLUDE_PATH}
-  /usr/local/include
-  /usr/include
-  )
-
-SET(_botan_LIBRARIES_SEARCH_DIRS
-  ${CMAKE_LIBRARY_PATH}
-  /usr/local/lib
-  /usr/lib
-  )
-
-##
-if( "${BOTAN_ROOT}" STREQUAL "")
```

Comment:
There is no need to use custom variables, because CMake provides the built-in ones. For example `CMAKE_FIND_ROOT_PATH` or `CMAKE_PREFIX_PATH`. More detailed description you can refer to [cmake official documentation](https://cmake.org/cmake/help/v3.6/command/find_path.html?highlight=find%20path#command:find_path)